docs/docs/installation/configuring-superset.mdx

```diff
@@ -69,6 +69,32 @@ you can add the endpoints to `WTF_CSRF_EXEMPT_LIST`:
 WTF_CSRF_EXEMPT_LIST = [‘’]
 ```
 
+### Using a production metadata database
```

Comment:
We've started to prefer the term "metastore", as "metadata database" is a bit of a tongue twister. But we can do a full update of that in a follow-up PR.